src/test/java/org/elasticsearch/transport/netty/NettyTransportMultiPortIntegrationTests.java

```diff
@@ -52,6 +57,8 @@ protected Settings nodeSettings(int nodeOrdinal) {
                 .put(TransportModule.TRANSPORT_TYPE_KEY, NettyTransport.class.getName())
                 .put("node.mode", "network")
                 .put("transport.profiles.client1.port", randomPortRange)
+                .put("transport.profiles.client1.publish_host", "10.0.254.253")
+                .put("transport.profiles.client1.publish_port", "4321")
```

Comment:
thats not a problem, as the publish address is just for publishing... which can be used, if you use port forwarding/docker in order to publish a different address than the one elasticsearch is really running on